main.js

```diff
@@ -1 +1,188 @@
+var buttonMeditate = document.querySelector('#meditate');
+var buttonExercise = document.querySelector('#exercise');
+var imageStudy = document.querySelector('.study-image');
+var imageMeditate = document.querySelector('.meditate-image');
+var imageExercise = document.querySelector('.exercise-image');
+var imageStudyActive = document.querySelector('.study-active');
+var imageMeditateActive = document.querySelector('.meditate-active');
+var imageExerciseActive = document.querySelector('.exercise-active');
+var userAccomplish = document.querySelector('#accomplish');
+var seconds = document.querySelector('#seconds');
+var minutes = document.querySelector('#minutes');
+var buttonStart = document.querySelector('#start');
+var form = document.querySelector('#activityForm');
+var clockForm = document.querySelector('#clockForm');
+var categoryError = document.querySelector('#buttonError');
+var minError = document.querySelector('#minutesError');
+var secError = document.querySelector('#secondsError');
+var accompError = document.querySelector('#accomplishError');
+var categoryButtons = document.querySelector('.button-box');
+var clockTime = document.querySelector('#clockTime');
+var clockAccomp = document.querySelector('#clockAccomp');
+var circlePurple = document.querySelector('.timer-meditate');
+var circleOrange = document.querySelector('.timer-exercise');
+var circleGreen = document.querySelector('.timer-study');
+seconds.addEventListener('blur', secondsError);
+minutes.addEventListener('blur', minuteError);
+userAccomplish.addEventListener('blur', accomplishError);
+categoryButtons.addEventListener('blur', catSelectionError);
+buttonMeditate.addEventListener('click', changeColorPurple);
+buttonStudy.addEventListener('click', changeColorGreen);
+buttonExercise.addEventListener('click', changeColorOrange);
+buttonStart.addEventListener('click', showActivity);
 
+var currentActivity = '';
+var previousActivities = [];
+
+
+function clock(accomp, min, sec) {
+  clockForm.classList.remove('hidden');
+  clockTime.innerText = `${min}:${sec}`;
+  clockAccomp.innerText = `${accomp}`;
+
+}
+
+function changeCircleColor(userCategory) {
+  if (userCategory === 'Exercise') {
+    circleOrange.classList.remove('hidden');
+  } else if (userCategory === 'Study') {
+    circleGreen.classList.remove('hidden');
+  } else if (userCategory === 'Meditate') {
+    circlePurple.classList.remove('hidden');
+  }
+}
+
+function showActivity() {
+var userCategory = categoryButtonFinder();
+var userDescription = userAccomplish.value;
+var userMinutes = minutes.value;
+var userSeconds = seconds.value;
+  if (userCategory && userDescription && userMinutes && userSeconds) {
+    form.classList.add('hidden');
+    currentActivity = new Activity(userCategory, userDescription, userMinutes, userSeconds)
+    clock(userDescription, userMinutes, userSeconds);
+    changeCircleColor(userCategory);
+  } else {
+    buttonError();
+  }
+}
```

Comment:
Looks like the showActivity function works even if incorrect statements are inputted into the minutes/seconds 

put random strings of characters into minutes and seconds, fill out the activity type, and click a button and it will bring you to the timer.

Reply:
Try it now.